ethers-contract/ethers-contract-abigen/src/multi.rs

```diff
@@ -333,7 +333,12 @@ impl MultiExpansionResult {
             None
         };
 
-        MultiBindingsInner { root, bindings, shared_types }
+        MultiBindingsInner {
+            root,
+            bindings,
+            shared_types,
+            dependencies: vec![String::from(r#"serde = "1.0.188""#)],
```

Comment:
hmm I don't think we serde by default here

what we need is a setter function just like `MultiBindings::format`

https://github.com/gakonst/ethers-rs/blob/a65abb57b2ee63470a9cf7c4fc98faeaaee8db53/ethers-contract/ethers-contract-abigen/src/multi.rs#L395-L402

so that this can be set manually:

`MultiBindings::dependencies(impl IntoIter<impl Into<String>) -> Self`